tidb-cloud/ticloud-config-create.md

```diff
@@ -0,0 +1,52 @@
+---
+title: ticloud config create
+Summary: The reference of `ticloud config create`.
+---
+
+# ticloud config create
+
+Create a [user profile](/tidb-cloud/cli-reference.md#user-profile) to store user profile settings:
+
+```shell
+ticloud config create [flags]
+```
+
+> **Note:**
+>
+> Before creating a user profile, you need to [create a TiDB Cloud API Key](https://docs.pingcap.com/tidbcloud/api/v1beta#section/Authentication/API-Key-Management).
```

Comment:
Other commands do not have this note. This note seems to be a general requirement for most of these commands and has been stated in [Before you begin](https://github.com/pingcap/docs/pull/12111/files#diff-be32acc0ba3f3660f7a8504bab797f03b905e299b628d776c4f2f97fc358313fR16) in get-started-with-cli. Do we need to add it here?

Reply:
Yes, the users may not read get-started-with-cli and see this page directly.

Reply:
Do other commands have this requirement as well? If so, maybe we should add this note for them, too.

Reply:
No, if the users use other commands without creating a profile, the CLI would remind them to create a profile, and then they can see the requirement of creating a user profile.